Frontend/ui-customer/src/Resources/Localizable/BN.purs

```diff
@@ -1349,4 +1349,6 @@ getBN stringKey =
         CLEAN_BIKE -> "পরিষ্কার বাইক"
         UNCOMFORTABLE_BIKE -> "অসুবিধাজনক বাইক"
         DRIVER_AVAILABLE -> "ড্রাইভার উপলব্ধ"
-        DRIVERS_AVAILABLE -> "ড্রাইভারগণ উপলব্ধ"
\ No newline at end of file
+        DRIVERS_AVAILABLE -> "ড্রাইভারগণ উপলব্ধ"
+        METRO_TICKETS -> "মেট্রো টিকেট"
+        EVERY_AMOUNT_ONE_FREE amount -> "প্রতি" <>" "<> amount <> "rd" <> " "<> "তে এক ফ্রি"
```

Comment:
@aryan010803 this `rd` also has to be sent as a parameter only as this can also change